src/main/java/com/microfocus/application/automation/tools/model/RunFromFileSystemModel.java

```diff
@@ -48,7 +49,11 @@
 
     public final static EnumDescription FAST_RUN_MODE = new EnumDescription("Fast", "Fast");
     public final static EnumDescription NORMAL_RUN_MODE = new EnumDescription("Normal", "Normal");
+    public final static EnumDescription ANY_BUILD_TEST = new EnumDescription("Of any of the build's tests", "Of any of the build's tests");
```

Comment:
You should create your model under uft folder and just introduce a filed inside your own model.